### ISSUE TITLE
Super-minimal Dockerfile setup

### ISSUE THREAD

**Issue by taiya** [envygeeks/jekyll-docker#315]
Hey @envygeeks I was trying to figure out a minimalistic way to reproduce the image that I executed as:
`docker run -v "$HOME/jalpc:/srv/jekyll" -p 4000:4000 -it jekyll/jekyll:3.5 jekyll serve`

AFAIK, all I need is a Dockerfile, and poking at your repo I found this:
https://github.com/envygeeks/jekyll-docker/blob/master/repos/jekyll/Dockerfile

However, it's littered with tokens like `<%= @meta.base_image %>`, which I assume then fetch the right configuration from `opts.yml`?:
https://github.com/envygeeks/jekyll-docker/blob/master/opts.yml

But what do you use to inject the yml file into the docker? ... I couldn't find it in your repo.
Basically I am trying to define the content of a Dockerfile so that I can just:

```
docker build -f Dockerfile -t myjekyll:latest .
docker run -v "$HOME/jalpc:/srv/jekyll" -p 4000:4000 -it myjekyll:latest jekyll serve
```

What you have done is great, but I'd like a self-contained setup :)
Thanks for the pointers

**Comment by rockstorm101** [comment]
I know this is an old issue but a key hint to the mysterious tokens lies in [docker-template][1]. In case it helps anyone.

[1]: https://github.com/envygeeks/docker-template

**Comment by envygeeks** [comment]
This should be fixed by the new image work.
